src/components/sde/tests/Simple2/simple2.hpp

```diff
@@ -0,0 +1,19 @@
+#if !defined(SIMP2_LIB_H)
```

Comment:
Instead of `#if !defined(SIMP2_LIB_H)` you could just use `ifndef PAPI_SIMPLE2_H`.